### DISCUSSION TITLE
this recipe doesn't follow the explanation explained below it in Workflows

### DISCUSSION THREAD

**Discussion by njbowen** [biostars/biostar-handbook-issues#323] [General]
make -f src/recipes/short-read-alignments.mk run

the reference accession is different, etc.

**Comment by njbowen** [comment]
I wasn't considering it a generic workflow that we could alter.  But now I see that we can in order to match what is below.

**Reply by ialbert** [reply]
I will look into making the information clearer, 

the same workflow is reused later in human snp calling, or some other guide, and I may have made it more generic then

**Reply by njbowen** [reply]
it's all ebola, it's just that the recipe short-read-alignments.mk is all 2014 sequence for both the reference and the reads, and the bam is labeled with the reference accession instead of the read accession.  
the bwa and bowtie examples below are using the older 1976 reference and a few more reads, so the SNPs are visible easier in IGV for teaching, thanks, great examples.  appreicate you, as always.

**Reply by ialbert** [reply]
That is a bug alright; the BAM file should be derived by the SRR number.

there is one ongoing issue/tension with the ebola example 

the snpEff default database for ebola is built  from the 2014 sequence accession KM233118 so anytime I want to to seamlessly integrate with snpEff I use KM233118 rather than the accession from 1976

sometimes those changes make it into examples and lead to inconsistencies, but in general all parameters are easily overwriteable

`ACC=KM233118` vs `ACC=AF086833` 

I often wonder if the defaults would be better off not set at all, and training students to learn to override these from the get go, 

in the end in their work they will have to override the parameters anyway

**Reply by njbowen** [reply]
thank you for that info.  my comment was just also general for the good of the cause, just in case anyone else ran across the same thing and went, hummm? .  and i was a little quick to point it out before realizing you meant it to be a modifiable template.  I agree, perhaps a set of recipes with defaults not set would be useful on many levels. In this case, I was guilty of using the newer workflow-makefiles and newer recipes you provide and not making sure that my students nor I had gone through their step-wise construction beforehand.  i have now defined all of the accessions, references, reads etc. wrt ebola, so they think about it a little more.  best,
